### ISSUE TITLE
dedupe dedupe --check reported as unlisted binary

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [webpro/knip#125]
In https://github.com/JoshuaKGoldberg/template-typescript-node-package/pull/393/commits/588a202a50e7e755e0c32dc2fd5591565c95bc74, I get:

```plaintext
$ pnpm lint:knip

> template-typescript-node-package@1.27.71 lint:knip /Users/josh/repos/template-typescript-node-package
> knip

Unlisted binaries (1)
dedupe  package.json
Configuration issues (1)
Unused item in ignoreBinaries: grep
 ELIFECYCLE  Command failed with exit code 1.
```

This comes from `package.json`:

```json
"scripts": {
	"lint:packages": "pnpm dedupe --check",
}
```

Looks like Knip doesn't recognize `dedupe` as a built-in command for `pnpm`?

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v2.10.3. See [Release 2.10.3](https://github.com/webpro/knip/releases/tag/2.10.3) for release notes._

**Comment by webpro** [comment]
Piew piew :)
